src/base/rss/rss_autodownloadrule.h

```diff
@@ -32,6 +32,10 @@
 #include <QSharedDataPointer>
 #include <QVariant>
 
+#include <boost/optional.hpp>
```

Comment:
Boost headers should be placed above Qt headers, according to the coding guidelines...